src/test/java/com/commercetools/sync/customobjects/CustomObjectSyncOptionsBuilderTest.java

```diff
@@ -86,54 +85,55 @@ void getThis_ShouldReturnCorrectInstance() {
     @Test
     void customObjectSyncOptionsBuilderSetters_ShouldBeCallableAfterBaseSyncOptionsBuildSetters() {
         final CustomObjectSyncOptions customObjectSyncOptions = CustomObjectSyncOptionsBuilder
-                .of(CTP_CLIENT)
-                .batchSize(30)
-                .beforeCreateCallback((newCustomObject) -> null)
-                .beforeUpdateCallback((updateActions, newCustomObject, oldCustomObject) -> emptyList())
-                .build();
+            .of(CTP_CLIENT)
+            .batchSize(30)
+            .beforeCreateCallback((newCustomObject) -> null)
+            .beforeUpdateCallback((updateActions, newCustomObject, oldCustomObject) -> emptyList())
+            .build();
```

Comment:
this section seems to be not indented properly. just verify :)

Reply:
I found that it is also indented like this in other SyncOptionsBuilderTest.